src/nekontrol/interactive/cli.py

```diff
@@ -22,15 +22,17 @@ def cli():
 @click.option("-c", "--color", type=bool)
 @click.option("-v", "--verbose", type=bool)
 @click.option("--ignore-debug", type=bool)
-def test_cli(
+def test(
     file_path: str,
-    problem: Optional[str],
-    color: Optional[bool],
-    diff: Optional[bool],
-    ignore_debug: Optional[bool],
-    verbose: Optional[bool],
+    problem: str | None,
+    color: bool | None,
+    diff: bool | None,
+    ignore_debug: bool | None,
+    verbose: bool | None,
 ):
     """Run and test against sample and local test data."""
+    setup_console(color)
```

Comment:
Maybe we could remove the color option, rich handles it well. Not for this pr though perhaps.

Reply:
eh i mean i added it here so might as well not add at instead :))